src/index.d.ts

```diff
@@ -369,6 +369,49 @@ declare namespace SDKStandardComponents {
             servicesBody: fspiopAPI.Schemas.ErrorInformationObject,
             destParticipantId: string
         ): Promise<GenericRequestResponse | GenericRequestResponseUndefined>;
+
+        /**
+         * @function postThirdpartyRequestsVerifications
+         * @description
+         *   Executes a `POST /thirdpartyRequests/verifications` request
+         * @param {Object} thirdpartyRequestsTransactionsBody The thirdpartyRequestsVerificationsBody
+         * @param {string} destParticipantId The id of the destination participant, in this case, a DFSP
+         * @returns {Promise<object>} JSON response body if one was received
+         */
+        postThirdpartyRequestsVerifications(
+            thirdpartyRequestsVerificationsBody: tpAPI.Schemas.ThirdpartyRequestsVerificationsPostRequest,
```

Comment:
I think you should address the schema changes needed for the fido assertion before merging this, you'll have to push another build after the changes other wise.

https://github.com/mojaloop/api-snippets/pull/101/files#r684868466

Reply:
Yeah I think you're right... I've been putting them off for to long 🤣

Reply:
Done!